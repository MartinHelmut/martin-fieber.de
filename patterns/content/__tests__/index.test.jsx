import React from "react";
import renderer from "react-test-renderer";

import Content from "../index";

describe("Content", () => {
  test("should render", () => {
    const tree = renderer.create(<Content>Much content</Content>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
