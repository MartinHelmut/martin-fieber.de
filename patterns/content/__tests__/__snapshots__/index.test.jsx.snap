// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render 1`] = `
<div
  className="content__Content-sc-1fjjc5a-0 hukOzd"
>
  Much content
</div>
`;
